extract_wheels/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""extract_wheels
```

Comment:
Uses the Yapf model of having main function in init.py so no magic has to be done with PYTHONPATHS